tools/deploy_assisted_installer_configmap.py

```diff
@@ -42,12 +42,13 @@ def main():
             print("Deploying {}".format(DST_FILE))
 
             versions = {"IMAGE_BUILDER": "installer-image-build",
-                        "AGENT_DOCKER_IMAGE": "agent",
                         "IGNITION_GENERATE_IMAGE": "assisted-ignition-generator",
+                        "KUBECONFIG_GENERATE_IMAGE": "ignition-manifests-and-kubeconfig-generate",
```

Comment:
@tsorya @yevgeny-shnaidman this seems a bit off.
Didn't we change the name?

Reply:
we did. KUBECONFIG_GENERATE_IMAGE is absolete as well as ignition-manifests-and-kubeconfig-generate

Reply:
@tsorya  can you please remove this line?